portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,12 +27,14 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private final List<String> greetings = Arrays.asList("I love salamanders","Macarons are my favorite food","How do you like these apples");
+  private final List<String> comments = new ArrayList<>();
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(greetings);
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String currcomment = getParameter(request, "comment", "");
+    comments.add(currcomment);
```

Comment:
You can combine these two lines if you want to and make
comments.add(getPrameter(request, "comment", "");

Reply:
I think I'm going to leave my syntax because I'm more comfortable having it broken down, it's easier for me to read